src/cdomains/lval.ml

```diff
@@ -9,13 +9,6 @@ type ('a, 'b) offs = [
   | `Index of 'b * ('a,'b) offs
 ] [@@deriving eq, ord]
 
-type ('a,'b) offs_uk = [
-  | `NoOffset
-  | `UnknownOffset
-  | `Field of 'a * ('a,'b) offs
-  | `Index of 'b * ('a,'b) offs
-]
```

Comment:
This was only used by the unused (and now removed) `Stateless` below.
Moreover, this contained a copy-paste error: it used `offs` instead of recursively using `offs_uk` like it should've. That's the reason why `Stateless` wasn't even correctly typed.